src/components/CheckTile/index.stories.js

```diff
@@ -0,0 +1,68 @@
+import React, {Component} from 'react'
+import { storiesOf } from '@storybook/react'
+import { withInfo } from '@storybook/addon-info'
+import shondaRhimesThumbnail from '../../images/shonda-rhimes-video.png'
+
+import CheckTile from '../CheckTile'
+
+const infoProps = {
+  inline: true,
+  header: false,
+};
+
+// CheckTile Handler
+class CheckTileHandler extends Component {
+  state = { checked: false }
+
+  render () {
+    const { checked } = this.state
+
+    return (
+      <div style={{ maxWidth: '500px' }}>
+        <CheckTile
+          {...this.props}
+          checked={checked}
+          onCheck={() => this.setState({ checked: !checked })}
+        >
+        </CheckTile>
+      </div>
+    )
+  }
+}
+
+
+storiesOf('CheckTile', module)
+  .add('CheckTile - 1000x609',
+    withInfo({ ...infoProps })(() =>
+      <div style={{ maxWidth: '500px' }}>
+        <CheckTile
+          imageUrl={shondaRhimesThumbnail}
+          checked={true}
```

Comment:
Omit the value of the prop when it is explicitly `true`.

Reference: https://github.com/airbnb/javascript/tree/master/react